extensions/wikia/Email/Controller/FounderDigestController.class.php

```diff
@@ -0,0 +1,312 @@
+<?php
+
+namespace Email\Controller;
+
+use Email\EmailController;
+use Email\Check;
+use Email\Tracking\TrackingCategories;
+
+abstract class FounderDigestController extends EmailController {
+	// Defaults; will be overridden in subclasses
+	const TRACKING_CATEGORY_EN = TrackingCategories::DEFAULT_CATEGORY;
+	const TRACKING_CATEGORY_INT = TrackingCategories::DEFAULT_CATEGORY;
+
+	protected $wikiId;
+
+	protected $pageViews;
+
+	protected $pageEdits;
+
+	protected $newUsers;
```

Comment:
Looks good. The last thing is I think it'd probably make more sense for the declarations for `$newUsers`, and `$pageEdits` to be in `FounderActivityDigestController` since that's the class which actually uses them.